clients/samba/samba.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Immediately abort the script on any error encountered
+set -e
+
+IP_ADDR=$(hostname -i | awk '{print $1}')
+FLAGS=""
+
+if [ "$HIVE_CLIENT_PRIVATE_KEY" != "" ]; then
+    FLAGS="$FLAGS  --unsafe-private-key=0x$HIVE_CLIENT_PRIVATE_KEY"
+fi
+
+ echo $HIVE_CLIENT_PRIVATE_KEY
+
+/opt/samba/bin/samba \
+    --jsonrpc-port=8545 \
+    --jsonrpc-host="${IP_ADDR}" \
+    --p2p-ip="${IP_ADDR}" \
+    --use-default-bootnodes=false \
+    --portal-subnetworks="history-network" \
+    $FLAGS
```

Comment:
please add a new line at the bottom of the file to remove 
![image](https://github.com/user-attachments/assets/f2c75a22-3c14-4d65-a00f-8e7c7af69de8)